### PR TITLE
{lib}[foss/2018a] PROJ v4.9.3

### PR DESCRIPTION
**Pull request by molden**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/p/PROJ/PROJ-4.9.3-foss-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+#
+# Copyright:: Copyright 2014-2015 The Cyprus Institute
+# Authors:: Thekla Loizou <t.loizou@cyi.ac.cy>
+# License:: MIT/GPL
+#
+##
+easyblock = 'ConfigureMake'
+
+name = 'PROJ'
+version = '4.9.3'
```
Comment (boegel) [inline easybuild/easyconfigs/p/PROJ/PROJ-4.9.3-foss-2018a.eb]: @molden Any reason you're not using the most recent `5.0.0`?

As far as I could tell from the release notes, it should be OK to use PROJ 5.0.0 as a drop-in replacement for 4.x (no breaking changes).

I tried this myself as a dependency to `GDAL`, and this worked fine...

Comment (boegel) [reply easybuild/easyconfigs/p/PROJ/PROJ-4.9.3-foss-2018a.eb]: @molden ping on bumping PROJ to 5.0.0?

**Comment by molden** [comment]
You can close this pull request as it is sort of duplicate see #6022

**Comment by boegel** [comment]
Thanks for pointing that out @molden!

**Pull request closed without merging**